src/supporting-data/citations/components/LiteratureCitation.tsx

```diff
@@ -37,6 +37,36 @@ const getLinkToAuthor = (author: string) => ({
   search: `query=lit_author:"${author}"`,
 });
 
+const WORM_BREEDERS_GAZETTE_URL = 'http://www.wormbook.org/wli/';
+// const ARVO_URL = 'http://iovs.arvojournals.org/article.aspx?articleid=';
+const PGR_URL = 'https://www.ebi.ac.uk/~textman/pgr-htdocs/pgr/';
```

Comment:
Could these go in `src/shared/config/externalUrls.ts`?

Reply:
In this case I'd rather have them localised with the component - reason for this is these are legacy, only concern a handful of publications and might be removed in the future.